src/hotspot/share/gc/g1/g1BarrierSet.cpp

```diff
@@ -148,9 +148,9 @@ void G1BarrierSet::on_thread_attach(Thread* thread) {
   assert(!queue.is_active(), "SATB queue should not be active");
   assert(queue.buffer() == nullptr, "SATB queue should not have a buffer");
   assert(queue.index() == 0, "SATB queue index should be zero");
-  // Can't assert that the DCQ is empty.  There is early execution on
-  // the main thread, before it gets added to the threads list, which
-  // is where this is called.  That execution may enqueue dirty cards.
+  G1DirtyCardQueue& dq = G1ThreadLocalData::dirty_card_queue(thread);
```

Comment:
Would prefer to use distinct names for the two kinds of queues here, i.e. `satb_queue` and `dirty_queue` for better distinction.

Reply:
Changing to "satbq" and "dirtyq".